pipeline/tasks/package_metadata_tasks.py

```diff
@@ -39,6 +39,8 @@ def execute(self, api_name, api_version, organization_name, output_dir,
         with open(package_defaults_yaml) as dep_file:
             package_defaults = yaml.load(dep_file)
 
+        package_type = api_name == 'common-protos' and 'grpc_common' or 'grpc_client'
```

Comment:
oh I think I finally figured it out, it's a fancy tertiary operation. 

In any case, I much prefer this to be in the artman config.